### ISSUE TITLE
Codemirror configuration

### ISSUE THREAD

**Issue by klauswr** [sinnlabs/zk-codemirror#12]
Is there a chance to get access to codemirror config options via `<codemirror/>` zul element by - let's say - a further parameter "`config=`" where one can enter e.g. `config="tabSize: 2"`.

**Comment by nsa-a1** [comment]
Yep it is possible. We will implement feature asap. 03.10.2022, 23:08, "Klaus Wriessnegger" ***@***.***>: Is there a chance to get access to codemirror config options via <codemirror/> zul element by - let's say - a further parameter "config=" where one can enter e.g. "tabSize: 2".—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by klauswr** [comment]
:-)

Am 04.10.2022 16:07 schrieb Peter Liverovsky ***@***.***>:

Yep it is possible. We will implement feature asap. 03.10.2022, 23:08, "Klaus Wriessnegger" ***@***.***>: Is there a chance to get access to codemirror config options via <codemirror/> zul element by - let's say - a further parameter "config=" where one can enter e.g. "tabSize: 2".—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
